autofs-5.1.4 - enable SIGUSR2 handling in rpm spec file

From: Ian Kent <raven@themaw.net>

Because the preferred usage is to leave busy mounts in place and
re-connect to them on start up the SIGUSR2 logic hasn't been used
or tested for a long time.

But it can be useful so enable it so any problems with it can be
identified and resolved.

It turns out that the SIGUSR2 forced shutdown behaves like a SIGUSR1
prune expire excpet that autofs will shutdown if all current mounts
are expired. But the kernel currently won't trigger expires for
mounts that are in use so it doesn't function quite as intended.

Enable the forced shutdown anyway so it will function if the kernel
supports it.

Signed-off-by: Ian Kent <raven@themaw.net>
---
 CHANGELOG   |    1 +
 autofs.spec |    1 +
 2 files changed, 2 insertions(+)

diff --git a/CHANGELOG b/CHANGELOG
index 0b7983ea..740f804f 100644
--- a/CHANGELOG
+++ b/CHANGELOG
@@ -29,6 +29,7 @@ xx/xx/2018 autofs-5.1.5
 - add-man page note about extra slashes in paths.
 - change expire type naming to better reflect usage.
 - use defines for expire type.
+- enable SIGUSR2 handling in rpm spec file.
 
 19/12/2017 autofs-5.1.4
 - fix spec file url.
diff --git a/autofs.spec b/autofs.spec
index f857d9da..6419e3e3 100644
--- a/autofs.spec
+++ b/autofs.spec
@@ -97,6 +97,7 @@ LDFLAGS="-Wl,-z,now" \
 ./configure --libdir=%{_libdir} \
 	--disable-mount-locking \
 	--enable-ignore-busy \
+	--enable-forced-shutdown \
 	%{?systemd_configure_arg:} \
 	%{?libtirpc_configure_arg:} \
 	%{?fedfs_configure_arg:}
